lightkurve/interact.py

```diff
@@ -388,22 +396,39 @@ def go_left_by_one():
             if existing_value > np.min(tpf.cadenceno):
                 cadence_slider.value = existing_value - 1
 
+        def export_mask():
+
+            if tpf_source.selected.indices != []:
+                selected_indices = np.array(tpf_source.selected.indices)
+                selected_mask = np.isin(pixel_index_array, selected_indices)
+                lc_new = tpf.to_lightcurve(aperture_mask=selected_mask)
+                lc_new.to_fits(exported_filename, overwrite=True,
+                               aperture_mask=selected_mask.astype(np.int),
```

Comment:
You're passing an aperture into a light curve fits file. This is not standard behavior. Is this going to confuse users between `LightCurveFiles` and custom light curve fits files? Can you read in a custom light curve fits file with light curve? Then how do you access the mask?

Maybe we should chat about this as a group?

Reply:
@christinahedges Some related discussions at #264 and to-a-lesser-extent at #265

Reply:
Following discussions, I think we decided to go ahead and accept `aperture_mask` as an argument to `to_fits`.

Note: it would be fair for `lc.to_fits` to accept both a boolean or an integer mask.  However in the case of the integer mask we should follow the standard Kepler/TESS bit convention.  In this current PR we pass `selected_mask.astype(np.int)` which will be incorrect (i.e. selected pixels will show value `1` instead of `3`).